### PR TITLE
BNF-1257 Fix redirects and URL paths

### PR DESCRIPTION
**Pull request by ediblecode**
https://nicedigital.atlassian.net/browse/BNF-1257

### CODE REVIEW

File: gatsby/plugins/gatsby-source-bnf/src/field-extensions/slug.test.ts [hunk 1]
```diff
@@ -31,6 +31,34 @@ describe("slug field extension", () => {
...
+			expect(
+				slugify("<span>Anti-D (Rh<sub>0</sub>) immunoglobulin</span>")
+			).toBe("anti-d-rh0-immunoglobulin");
+		});
+
+		it("should retain letter number combos", () => {
+			expect(slugify("C1-esterase inhibitor")).toBe("c1-esterase-inhibitor");
+		});
+
+		it("should replace apostrophes with hyphens", () => {
```
Comment (johndavey72) [inline gatsby/plugins/gatsby-source-bnf/src/field-extensions/slug.test.ts:45]: description doesn't reflect the assertion.... I would expect parkinson-s-disease based on that desc

Comment (ediblecode) [reply gatsby/plugins/gatsby-source-bnf/src/field-extensions/slug.test.ts:45]: Good spot, will sort!

Comment (ediblecode) [reply gatsby/plugins/gatsby-source-bnf/src/field-extensions/slug.test.ts]: Sorted!

File: gatsby/plugins/gatsby-source-bnf/src/field-extensions/slug.test.ts [hunk 2]
```diff
@@ -31,6 +31,34 @@ describe("slug field extension", () => {
...
+			expect(slugify("Parkinson's disease")).toBe("parkinsons-disease");
+		});
+
+		it("should replace curly apostrophes with hyphens", () => {
```
[the remark this answers is not in the document]
Comment (ediblecode) [reply gatsby/plugins/gatsby-source-bnf/src/field-extensions/slug.test.ts]: Sorted!

**Review by johndavey72 — APPROVED** [review]

**Pull request merged**